storybook/src/admin-date-time/AllDateAndTimePickers.tsx

```diff
@@ -69,4 +68,8 @@ const Story = () => {
     );
 };
 
-storiesOf("@comet/admin-date-time", module).add("All Pickers", () => <Story />);
+export default {
+    title: "@comet/admin-date-time",
+};
+
+export const AllPickers = () => <Story />;
```

Comment:
Wouldn't it be simpler to just export the story directly, instead of a function that renders the story?

Reply:
Changed: 85c5eefe5c5cba41c3b4cbcfb02904108476070b